closed/get_openssl_source.sh

```diff
@@ -26,6 +26,7 @@ usage() {
 	echo "Usage: $0 [-h|--help] [--openssl-version=<openssl version 1.0.2 and above to download>]"
```

Comment:
Perhaps this should change; either mention the new option or just say `[options]` or similar.